### DISCUSSION TITLE
[Question][Hw0-2] Am I allowed to add my tests in test/*.ml?

### DISCUSSION THREAD

**Discussion by tncks0121** [prosyslab-classroom/cs348-information-security#648] [Q&A]
Name: Suchan Park

The readme says that:

> Preliminary examples in the document are provided as a suite of unit test (`test/[Problem name]Test.ml`).

Suppose HW0-2 is an actual graded homework. In that case, am I allowed to add some tests in that file, or should I create another file for my tests? 

For example, homework.pdf contains some examples, and I want to add them explicitly in the test suite.

**Comment by yeonhee-ryou** [comment]
We stated in the README that

> students are not permitted to change directory structures

However, we will only grade the code you committed and pushed.

**Comment by tncks0121** [comment]
Ok, thanks for the clarification
